### DISCUSSION TITLE
Did something change with notifications?

### DISCUSSION THREAD

**Discussion by rnjvbox** [andrei-tatar/node-red-contrib-smartnora#246] [General]
Hi,

Some months ago I had tested with notifications and it worked perfectly. I worked on some other projects before I wanted to start with the notifications now. When I went back to my test setup I noticed that it wasn't working anymore. And when I wanted to re-subscribe to receive the notifications I got on my phone (Samsung with Android app) the message "Could not subscribe to receive notifications". On my PC (Windows 11 with Chrome) I got the message subscribed successfully, but still don't receive notifications. In Node Red all seems to work fine, status change to "sent" and back to "connected" without errors in de debug screen.

Kind regards,
René

**Comment by andrei-tatar** [comment]
Hi @rnjvbox, it seems like the service worker scripts were way of of date. I updated them and pushed the change. 
Can you please try it now?

When going to https://smart-nora.eu/my-nora , wait a few seconds and it will tell you there's an update available. Once the update is done, it should work fine again.

**Reply by rnjvbox** [reply]
Hi Andrei,

I indeed got the message that there was a refresh and after reload the page the notification was working again.

Thanks for the great support!

Kind regards,
René
